Carbon.FileSystem/Carbon.FileSystem.psd1

```diff
@@ -78,7 +78,11 @@
 
     # Functions to export from this module. Only list public function here.
     FunctionsToExport = @(
-        'Get-FileHardLink'
+        'Get-CNtfsPermission',
+        'Get-FileHardLink',

```

Comment:
Why does `Get-FileHardLink` not contain the `CNtfs` prefix?

Reply:
Will rename.

Reply:
It was like this when I got here.